### ISSUE TITLE
Response  body not showing when using Illuminate\Http\Response constants.

### ISSUE THREAD

**Issue by vszambon1** [dedoc/scramble#169]
Hey,

When i use **Illuminate\Http\Response** HTTP status codes for better readability, the doc stops showing the response body.

`response()->json(['message' => 'Foo Bar'], Response::HTTP_OK)`
![image](https://github.com/dedoc/scramble/assets/5924225/77e3629f-d996-4031-8cd1-825d9da350bc)

vs

`response()->json(['message' => 'Foo Bar'], 200)`
![image](https://github.com/dedoc/scramble/assets/5924225/9cedbccf-4864-46ad-bb30-9b28185151f7)

**Comment by romalytvynenko** [comment]
@vszambon1 true! Thanks for reporting. I also can see that response structure in general is not correctly documented in such case? Because expected structure is array, `{message: string}`, not just string.

Will fix!

**Comment by vszambon1** [comment]
> @vszambon1 true! Thanks for reporting. I also can see that response structure in general is not correctly documented in such case? Because expected structure is array, `{message: string}`, not just string.
> 
> Will fix!

Actually the code that I used to generate the print didn't had the 'message' key, sorry for that.

**Comment by romalytvynenko** [comment]
Oh, okay. So, I'll fix only constants 👾
